frontend-maven-plugin/src/main/java/com/github/eirslett/maven/plugins/frontend/mojo/AbstractFrontendMojo.java

```diff
@@ -23,13 +35,23 @@
   @Parameter(defaultValue = "${basedir}", property = "installDirectory", required = false)
   protected File installDirectory;
 
+  /**
+   * Determines if this execution should be skipped.
+   */
+  private boolean skipTestPhase() {
+    return skipTests && execution.getLifecyclePhase().equals("test");
```

Comment:
Would it also be worth adding a check for "integration-test" lifecycle phase too?